rust/basic_bitcoin/src/basic_bitcoin/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+TARGET="wasm32-unknown-unknown"
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+pushd $SCRIPT_DIR
+
+# NOTE: On macOS a specific version of llvm-ar and clang need to be set here.
+# Otherwise the wasm compilation of rust-secp256k1 will fail.
+if [ "$(uname)" == "Darwin" ]; then
+  # On macs we need to use the brew versions
```

Comment:
So, the documentation needs to point out that Mac users need `brew` installed, right?

Reply:
Yes, exactly. We had this documentation as part of the dev preview, and we will have it on our portal documentation as well.